xchange-binance/src/main/java/org/knowm/xchange/binance/BinanceErrorAdapter.java

```diff
@@ -45,15 +45,16 @@ public static ExchangeException adapt(BinanceException e) {
         return new CurrencyPairNotValidException(message, e);
       case -1122:
         return new ExchangeSecurityException(message, e);
+      default:
+        return new ExchangeException(message, e);
     }
-    return new ExchangeException(message, e);
   }
 
-  private static ExchangeException createOrderNotValidException(
-      String message, BinanceException e) {
+  private static ExchangeException createOrderNotValidException(String message, BinanceException e) {
     if (e.getMessage().contains("MIN_NOTIONAL")) {
       return new OrderAmountUnderMinimumException(message, e);
+    } else {
```

Comment:
You don't need an else statement here. Even though both methods are correct it comes down to readability by reducing nesting in your code.

Reply:
I aligned this to the same convention as few lines above:

```
 if (e.getMessage().contains("insufficient balance")) {
          return new FundsExceededException(e.getMessage(), e);
 } else {
          return new ExchangeException(message, e);
 }
```